.pylintdict

```diff
@@ -88,6 +88,8 @@ maxiter
 mcrx
 mcry
 mcrz
+mixin
+misclassified
```

Comment:
Ideally these would be in swapped to preserve alphabetic ordering

Reply:
> Ideally these would be in swapped to preserve alphabetic ordering

Fixed